src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -254,16 +394,78 @@ unique_ptr<CatalogEntry> TableCatalogEntry::AddColumn(ClientContext &context, Ad
 	                                      new_storage);
 }
 
-unique_ptr<CatalogEntry> TableCatalogEntry::RemoveColumn(ClientContext &context, RemoveColumnInfo &info) {
+void DependsOnColumn(const ParsedExpression &expr, const string &column, bool &result) {
```

Comment:
Can `result` be turned into a return value?

Reply:
Ideally yes, I dont quite know an elegant way how though, other than adding another function inbetween
Since it's used together with ParsedExpressionIterator there isn't a way to return a value from DependsOnColumn directly

Reply:
I've opted for the "other function inbetween" option for now, I would be interested to know how else I could fix it 👍